cypress/e2e/personal-tokens.cy.js

```diff
@@ -0,0 +1,82 @@
+const selectors = {
+  menu: {
+    userAvatar: '[data-testid="profile-block__avatar"] > .p-avatar-icon',
+    personalTokenItem: 'li[aria-label="Personal Token"] > .p-menuitem-content > .p-menuitem-link'
+  },
+  list: {
```

Comment:
Thinking about the possibility of the selectors being merged in the future, I suggest that you rename the list and form entries to _networkListList_ and _networkListEditForm_ or _networkList.list_ and _networkList.editForm_